### PR TITLE
[qml] Remove PageStackAction.Replace for about:config dialog. Fixes JB#32873

### PR DESCRIPTION
**Pull request by rojkov**
…B32873

### CODE REVIEW

File: src/pages/components/ConfigWarning.qml [hunk 1]
```diff
@@ -19,7 +19,6 @@ ConfirmDialog {
     //: Warning of changing browser configurations.
     //% "Changing these advanced settings can cause issues with stability, security and performance of Sailfish Browser. Continue ?"
     text: qsTrId("sailfish_browser-la-config-warning");
-    acceptDestinationAction: PageStackAction.Replace
```
Comment (siteshwar) [inline src/pages/components/ConfigWarning.qml]: Sounds like more of a workaround than actual fix.

Comment (rojkov) [inline src/pages/components/ConfigWarning.qml]: Yes, it is. There's a bug in Silica.

**Comment by siteshwar** [comment]
LGTM

File: src/pages/components/ConfigWarning.qml (hunk 1 above)
Comment (rainemak) [inline src/pages/components/ConfigWarning.qml]: As this fixes bigger issue LGTM.

**Pull request merged**